spec/features/course_cloning_spec.rb

```diff
@@ -104,5 +105,6 @@
     expect(new_course.submitted).to eq(false)
     expect(new_course.user_count).to be_zero
     expect(new_course.article_count).to be_zero
+    expect(new_course.wikis.count).to eq(1)
```

Comment:
I think it would be better to extend this test so that the original course has more than one wiki, and make sure the set of tracked wikis is cloned as expected.